File: lib/_satis/saatlik_satis/model/saatlik_satis_model.dart
import 'dart:convert';

import '../../../core/base/base_model.dart';
import 'package:json_annotation/json_annotation.dart';

part 'saatlik_satis_model.g.dart';

@JsonSerializable()
class SaatlikSatisModel extends BaseModel {
  final DateTime? tarih;
  final String? saat;
  final int? musSay;
  final double? sepetOrt;
  final double? ciro;
  final double? yemekKarti;
  final double? toplamSepet;

  SaatlikSatisModel(this.tarih, this.saat, this.musSay, this.sepetOrt, this.ciro, this.yemekKarti, this.toplamSepet);

  Map<String, dynamic> toJson() => _$SaatlikSatisModelToJson(this);
  factory SaatlikSatisModel.fromJson(Map<String, dynamic> json)=>
      _$SaatlikSatisModelFromJson(json);  
  @override
  String myJson() => jsonEncode(toJson());
}

File: GiroDashboard.Application/DTOs/Satis/SaatlikSatisModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GiroDashboard.Application.DTOs.Satis
{
    public class SaatlikSatisModel
    {
        public DateTime? Tarih { get; set; }

        public string? Saat { get; set; }    

        public int? MusSay { get; set; }

        public decimal? SepetOrt { get; set; }

        public decimal? Ciro { get; set; }

        public decimal? YemekKarti { get; set; }

        public decimal? ToplamSepet { get; set; }
    }
}
